docker/rust/Dockerfile

```diff
@@ -1,5 +1,5 @@
-# With this image, only the naive solver will work
-ARG SOLVER_BASE=ubuntu:bionic
+# With this image, only the naive and open solver will work
+ARG SOLVER_BASE=gnosispm/dex-open-solver:v0.0.9-dev
```

Comment:
I would slightly prefer to keep the original base image. This would also make the commands of public/private solver more similar as both would require the SOLVER_BASE argument.